### ISSUE TITLE
Minor issue: missing description for diagnosis history sheet

### ISSUE THREAD

**Issue by GSMPRANEETH** [coronasafe/care_fe#14112]
**Describe the bug**
For Diagnosis History sheet, there is a console message about missing descriptin/aria-describedby

**To Reproduce**

1. Go to 'Patient'
2. Click on 'Diagnosis'
3. Scroll down to 'Diagnosis History'
4. See error in console

**Expected behavior**
No such error messages should be shown in console

**Screenshots**

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/1e9bf657-25b7-4afc-8917-110d2d86e045" />

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

---

### Instructions for Requesting Assignment:

To request assignment, please clearly outline your solution and timeline by commenting on the issue using the format below:

**Describe your solution clearly:**
Provide a detailed explanation of your proposed solution, including your approach, key implementation steps, and relevant examples or references. Mention any dependencies, assumptions, or risks you foresee that might affect your timeline or implementation.

**Expected Timeline:**
- End date: [Expected submission date of a completed Pull Request]

**Additional Context:**
Include any other relevant context, links, screenshots, or resources that support your proposed solution.

> 🚨 Your assignment may be unassigned if there is no activity or progress within the stated timeline unless communicated clearly and agreed upon.

**Comment by GSMPRANEETH** [comment]
@Jacobjeevan @rithviknishad @yash-learner @SiddanthNayak @abhimanyurajeesh 
I will add `SheetDescription` for the Diagnosis History component
ETA: EOD
PS: This error might occur for other various components that use Sheet/ Card, I will implement description where it is missing, this might take me a bit longer then ETA

**Comment by GSMPRANEETH** [comment]
Also occurs in Add/Edit Medication Form

**Comment by bodhish** [comment]
Assigning to @SiddanthNayak as he is already working on a PR related to this
